db/migrate/20231127115503_create_properties.rb

```diff
@@ -8,10 +8,10 @@ def change
       t.integer :area
       t.integer :number_of_rooms
       t.integer :number_of_bedrooms
-      t.boolean :furnished
-      t.boolean :terrace
-      t.boolean :basement
-      t.boolean :renting
+      t.boolean :furnished, default: true
```

Comment:
should we put a default value on those ones ?
it feels like we should make it non-null or at least on false

Reply:
Why this specific one? Shouldn't I then just put defaults on all of them?